src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -141,6 +146,9 @@ m_cursorGrabbed   (false)
 
     if (m_handle)
     {
+        if (handleCount++ == 0)
```

Comment:
I personally am not a big fan of this notation, but maybe others don't have an issue with it.